lib/vanagon/platform/windows.rb

```diff
@@ -289,33 +297,44 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Generate the root wix directory structure by compiling
+      # a list of items required then generating the structure
+      # based on that list
+      #
+      # @param services, list of all services in a project
+      # @param project, actual project we are creating the directory structure for
+      def generate_wix_dir_structure(services, project)
+        items = []
+        # All service files will need a directory reference
+        services.map { |svc| items.push({ :path => File.dirname(svc.service_file.gsub(/\\/, "/")), :element_to_add => "<Directory Id=\"#{svc.bindir_id}\" />\n" }) }
+        # INSTALLDIR is the root installation directory for the project
+        items.push({ :path => "SourceDir/#{project.settings[:base_dir]}/#{project.settings[:company_id]}/#{project.settings[:product_id]}", :element_to_add => "<Directory Id=\"INSTALLDIR\" />\n" })
```

Comment:
```:company_id``` should be ```:company_name```

You also need to make sure that the ```#{project.settings[:base_dir]}``` does not use the ```"Name"``` field as this cause the ```ProgamFiles64Folder``` directory to be created.

I think it would be better to actually express these directories directly in the Wix - see https://github.com/puppetlabs/vanagon/pull/318/files#r56167505 below